docs/docs/30-how-to-guides/10-installing-kargo.md

```diff
@@ -74,7 +74,7 @@ following:
 1. Edit and save the values.
 
    :::info
-   You will find this configuration file contains helpful comments for every
+   You will find that this configuration file (kargo-values.yaml) contains helpful comments for every
```

Comment:
This change is really unnecessary. They saw the filename in the previous step and they have it open and are already looking at it in this step. There's no need to clarify what file we're talking about.

Reply:
This is also unrelated to the subject of the PR.